pyproject.toml

```diff
@@ -1,5 +1,157 @@
+# listed.
+maintainers = [
+  {name = "A. Great Maintainer", email = "maintainer@example.com" } # Optional
+]
+
+# Classifiers help users find your project by categorizing it.
+#
+# For a list of valid classifiers, see https://pypi.org/classifiers/
+classifiers = [  # Optional
+  # How mature is this project? Common values are
+  #   3 - Alpha
+  #   4 - Beta
+  #   5 - Production/Stable
+  "Development Status :: 3 - Alpha",
+
+  # Indicate who your project is intended for
+  "Intended Audience :: Developers",
+  "Topic :: Software Development :: Build Tools",
+
+  # Pick your license as you wish
+  "License :: OSI Approved :: MIT License",
+
+  # Specify the Python versions you support here. In particular, ensure
+  # that you indicate you support Python 3. These classifiers are *not*
+  # checked by "pip install". See instead "python_requires" below.
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.6",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3 :: Only",
+]
+
+# This field lists other packages that your project depends on to run.
+# Any package you put here will be installed by pip when your project is
+# installed, so they must be valid existing projects.
+#
+# For an analysis of this field vs pip's requirements files see:
+# https://packaging.python.org/discussions/install-requires-vs-requirements/
+dependencies=[
+  "peppercorn"  # Optional
+]
+
+# List additional groups of dependencies here (e.g. development
+# dependencies). Users will be able to install these using the "extras"
+# syntax, for example:
+#
+#   $ pip install sampleproject[dev]
+#
+# Similar to `install_requires` above, these must be valid existing
+# projects.
+[project.optional-dependencies]
+dev = ["check-manifest"] # Optional
+test = ["coverage"] # Optional
+
+# List URLs that are relevant to your project
+#
+# This field corresponds to the "Project-URL" and "Home-Page" metadata fields:
+# https://packaging.python.org/specifications/core-metadata/#project-url-multiple-use
+# https://packaging.python.org/specifications/core-metadata/#home-page-optional
+#
+# Examples listed include a pattern for specifying where the package tracks
+# issues, where the source is hosted, where to say thanks to the package
+# maintainers, and where to support the project financially. The key is
+# what's used to render the link text on PyPI.
+[project.urls]  # Optional
+"Homepage" = "https://github.com/pypa/sampleproject"
+"Bug Reports" = "https://github.com/pypa/sampleproject/issues"
+"Funding" = "https://donate.pypi.org"
+"Say Thanks!" = "http://saythanks.io/to/example"
+"Source" = "https://github.com/pypa/sampleproject/"
+
+# To provide executable scripts, use entry points in preference to the
+# "scripts" keyword. Entry points provide cross-platform support and allow
```

Comment:
To avoid confusion with `project.scripts`, setuptools uses `tool.setuptools.script-files` for the legacy kind of script (which is [explicitly deprecated in the docs now](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration)).

Maybe we can just skip this advice for the sake of avoiding confusion?
PEP 621 splits general entry-points, `console_scripts` and `gui_scripts` in 3 separated fields. There is a chance the wording *entry point* might be confusing when referring to `scripts` for new pacakgers.

Reply:
Yes, this whole comment needs either re-wording or removing.

We shouldn't be mentioning the setuptools-specific "scripts" keyword at all here.